packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -208,6 +208,7 @@ export type CallAdapterClientState = {
     endedCall?: CallState;
     isTeamsCall: boolean;
     latestErrors: AdapterErrors;
+    isCameraOn?: 'On' | 'Off';
```

Comment:
Would be good to name this to cameraStatus - otherwise ‘On’ | 'Off' would be still only options for this